benchmarks/django_simple/gunicorn.conf.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+
+def post_fork(server, worker):
+    # Set lower defaults for ensuring profiler collect is run
+    if os.environ.get("PERF_ENABLE_PROFILER") == "1":
+        os.environ.update(
+            {"DD_PROFILING_ENABLED": "1", "DD_PROFILING_API_TIMEOUT": "0.1", "DD_PROFILING_UPLOAD_INTERVAL": "10"}
+        )
+    # This will not work with gevent workers as the gevent hub has not been
+    # initialized when this hook is called.
+    if os.environ.get("PERF_ENABLE_TRACER") == "1":
+        import ddtrace.bootstrap.sitecustomize  # noqa
+
+
+def post_worker_init(worker):
+    # If profiling enabled but not tracer than only run auto script for profiler
+    if os.environ.get("PERF_ENABLE_PROFILER") == "1" and os.environ.get("PERF_ENABLE_TRACER") == "0":
+        import ddtrace.profiling.auto  # noqa
+
+
+bind = "0.0.0.0:8000"
+if os.environ.get("PERF_THREADS"):
+    worker_class = "gthread"
+    threads = 10
```

Comment:
worth making num threads configurable too? maybe just reuse `PERF_THREADS` and treat > 0 as number of threads?